activesupport/lib/active_support/cache/strategy/local_cache.rb

```diff
@@ -132,7 +132,7 @@ def read_multi_entries(keys, **options)
 
             local_entries = local_cache.read_multi_entries(keys)
             local_entries.transform_values! do |payload|
-              deserialize_entry(payload).value
+              deserialize_entry(payload)&.value
```

Comment:
```suggestion
              deserialize_entry(payload).value unless payload.nil?
```

if I'm not mistaken, `payload` is `nil` here, so might as well skip `deserialize_entry`

Reply:
Actually nevermind, design wise it's best not to assume anything about how a nil payload may be deserialized.